app/account/forms.py

```diff
@@ -1,10 +1,9 @@
-from flask import url_for
 from flask.ext.wtf import Form
 from wtforms.fields import (
-    StringField,
     PasswordField,
     BooleanField,
```

Comment:
You can't see the entire file in this diff. `BooleanField` is used in `LoginForm`. 

As a code reviewer, you don't need to worry about unnecessary import statements. `flake8` checks to make sure that all imports are actually being used. :)